gnarly.rb

```diff
@@ -270,6 +276,21 @@ def setup_readme
   gsub_file "README.md", "__application_name__", app_name
 end
 
+def setup_i18n
+  missing_translation_config = <<-CONFIG
+
+  # Raise error when missing i18n translations
+  config.action_view.raise_on_missing_translations = true
```

Comment:
I *believe* this should already be in these files, but commented out.

Reply:
It is! I thought that was the case but must have missed it when I searched for it before. I'll change this to just `gsub` the comment out instead.